docs/UserGuide.md

```diff
@@ -87,21 +87,24 @@ Format: `help`
 
 ### Adding a guest: `add`
 
-Adds a guest to the guest book.
+Adds a guest to the guest book. This is akin to checking a guest in with their personal and room details.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS [rq/REQUEST]`
 
-[//]: # (Commented out first as not relevant since we do not have optional fields)
-[//]: # (But may be relevant in the future)
-[//]: # (<div markdown="span" class="alert alert-primary">:bulb: **Tip:**)
+Constraints of parameters:
 
-[//]: # (A guest can have any number of tags &#40;including 0&#41;)
-
-[//]: # (</div>)
+Unless stated otherwise, all fields should not be left blank.
+* `NAME` should only contain alphanumeric characters and spaces.
+* `PHONE_NUMBER` should only contain numbers, and it should be at least 3 digits long.
+* `EMAIL` should only contain alphanumeric characters and '@' character.
+* `ROOM` should only contain alphanumeric characters and a hyphen.
+* `DATE_RANGE` should follow the format dd/MM/yy - dd/MM/yy.
+* `NUMBER_OF_GUESTS` should only contain numbers, and it should be at least 1 and at most 4.
+* `REQUEST` is an optional field.
```

Comment:
Should we add that there can at most be 500 characters?

Reply:
Yes, nice catch!